### ISSUE TITLE
[accessibility] The × character in the close button of popups should be explicitly hidden from ATs

### ISSUE THREAD

**Issue by Malvoz** [mapbox/mapbox-gl-js#11035]
Although the contents of `aria-label` overrides the × character in the close button of popups, it is still announced as <q>multiplication</q> or <q>times</q> (depending on the screen reader) when the user is _hovering_ the control with a mouse pointer.

To solve this issue, the × character:

https://github.com/mapbox/mapbox-gl-js/blob/9c1e1a0364bc5f94f225f6d6246104422b6894e3/src/ui/popup.js#L537

should be wrapped in `<span aria-hidden="true">`.

(This would also be more consistent with other controls that hide icons from ATs, such as the zoom in control)

https://github.com/mapbox/mapbox-gl-js/blob/36533f3683dd48837ab07095fb6484165d9ee400/src/ui/control/navigation_control.js#L64
